QueenCam/QueenCam/Presentation/Views/Camera/CameraView.swift

```diff
@@ -25,6 +25,7 @@ struct CameraView {
   @State private var focusLocation: CGPoint = .zero
 
   @State private var isShowPhotoPicker = false
+  @State private var referenceVM = ReferenceViewModel()
```

Comment:
```suggestion
  @State private var referenceViewModel = ReferenceViewModel()
```

다른 코드들 볼 때 `~~ViewModel`로 적는게 일관적일 것 같은데 어떠신가용?